### ISSUE TITLE
Sign + ZipAlign

### ISSUE THREAD

**Issue by sbatezat** [SamVerschueren/gulp-cordova-build-android#12]
As per many documentation (see http://stackoverflow.com/questions/26449512/how-to-create-singed-apk-file-using-cordova-command-line-interface for example), to build an android app in release mode you need to sign it and zip align it. Maybe I'm wrong but I've got the feeling you are not zip align it. Are you?

If not, can you automatically add this after signing the app?

Many thanks
Sébastien

**Comment by SamVerschueren** [comment]
Hi Sébastien,

I thought the `--release` flag did that for you, but I can't seem to find any trace of my assumption so probably it isn't. Thanks for reporting this, will have a look at it.

**Comment by sbatezat** [comment]
Hi Sam,

I'm sure it isn't 'cause it's useless to zip align an APK before signing it, and the --release flag from Cordova is not signing the APK. Other reference I'm using : http://ionicframework.com/docs/guide/publishing.html

**Comment by SamVerschueren** [comment]
`--release` in combination with `release-signing.properties` is. So cordova can sign the apk for you, so in theory it can zipalign it as well :). But you are right, will make it happen ;). Not sure when though.

**Comment by SamVerschueren** [comment]
@sbatezat I had to upload an APK today to the store and used `gulp-cordova` to build the APK. I used the properties `storeFile`, `keyAlias`, `storePassword` and `keyPassword` to automatically sign the APK. The plugin generates an unaligned and an aligned version of the apk which means that it zipaligns the APK itself. So I'm not sure if this should be added manually or am I missing something here?
